content/MoM/mom1e/tiles.ini

```diff
@@ -24,6 +36,11 @@ pps=146.28571
 [TileSideFrontPorch]
 image="../ffg/img/Tile_Front_Porch_MAD01.dds"
 
+[TileSideFrontYard]
+image="img/Tile_Front_Yard_MAD01.jpg"
+aspect=1.5
```

Comment:
Ok, I did this way in order to fit exactly with the ffg ones. But it's true that in original ffg app, big tiles looks weird.
I will rescan it